[Doc] Modify transaction doc. (#10777)
apache/pulsar: site2/docs/transaction-api.md

---
id: transactions-api
title: Transactions API
sidebar_label: Transactions API
---

> All messages in a transaction is available only to consumers after the transaction has been committed. If a transaction has been aborted, all the writes and acknowledgments in this transaction rollback. 

Reviewer: ```suggestion
All messages in a transaction are available only to consumers after the transaction has been committed. If a transaction has been aborted, all the writes and acknowledgments in this transaction roll back. 
```
